src/wp-admin/includes/translation-install.php

```diff
@@ -35,23 +34,27 @@ function translations_api( $type, $args = null ) {
 	$res = apply_filters( 'translations_api', false, $type, $args );
 
 	if ( false === $res ) {
-		$url = $http_url = 'https://api.wordpress.org/translations/' . $type . '/1.0/';
-
+		$stats = array(
+			'wp_version' => $cp_version,
```

Comment:
Wondering why we need to send the cp_version to wordpress.org servers, maybe is better to leave them to a defalt `4.9.9`

Reply:
i'm wondering that too but since is we are supporting `4.9.8` and `4.9.9` and this is for stats only i guess we can leave it dynamic. Am i right?

Reply:
I think that we can leave as default `4.9.9` after all CP is based on that version.

Reply:
ok, done with last commit